e2e/cypress/tests/integration/ad_ldap/saml_ldap_sync_id_attrib_spec.ts

```diff
@@ -72,13 +72,13 @@ describe('AD / LDAP', () => {
         const randomId = getRandomId();
         const newFirstName = `Firstname${randomId}`;
         const newLastName = `Lastname${randomId}`;
-        cy.updateLDAPUser({
+        (cy as any).updateLDAPUser({
             ...samlLdapUser,
             firstname: newFirstName,
             lastname: newLastName,
         });
         const admin = getAdminAccount();
```

Comment:
we also need to type this function correctly in 
https://github.com/mattermost/mattermost-webapp/blob/f3ecda80ea47ca2645b4df5b387280a1541cb9d3/e2e/cypress/tests/support/env.ts#L10-L16

as

```ts
export function getAdminAccount() {
    return {
        username: Cypress.env('adminUsername'),
        password: Cypress.env('adminPassword'),
        email: Cypress.env('adminEmail'),
    } as Cypress.UserProfile;
}
```

Reply:
updated